onnx/defs/quantization/defs.cc

```diff
@@ -101,4 +102,50 @@ ONNX_OPERATOR_SET_SCHEMA(
               updateOutputShape(ctx, 0, input_shape);
             }));
 
+static const char* DynamicQuantizeLinear_ver11_doc = R"DOC(
+A Function to fuse calculation for Scale, Zero Point and FP32->8Bit convertion of FP32 Input data.
+Outputs Scale, ZeroPoint and Quantized Input for a given FP32 Input)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    DynamicQuantizeLinear,
+    11,
+    OpSchema()
+        .SetDoc(DynamicQuantizeLinear_ver11_doc)
+        .Input(0, "x", "Input tensor", "T1")
+        .Output(0, "y", "Quantized output tensor", "T2")
+        .Output(1, "y_scale", "Output Scale. It's a scalar or a 1D tensor with size 1.", "tensor(float)")
+        .Output(2, "y_zero_point", "Output Zero point. It's a scalar or a 1D tensor of size 1.", "T2")
```

Comment:
Ideally, we should say "scalar, that is a 0D tensor" in the preceding two lines. Is there any reason we need to say "or 1D tensor"? Let us try to uniformly use tensors with rank 0 for scalars.

Reply:
I added 1D tensor of size 1 to relax the requirement for y_scale and y_zero_point to be "Scalars" in onnx terms... the requirement is really that they be of size 1 .i.e 1 scale and zero point per layer.